lib/src/widgets/table_picker.dart

```diff
@@ -1 +1,276 @@
+        height: widget.iconSize,
+        child: ElTooltip(
+          color: widget.dropDownColor,
+          distance: 0,
+          onTap: () {
+            if (MediaQuery.of(context).size.width < 480) {
+              _showEditTableSheet(true, context);
+            } else {
+              if (_editTableETKey.currentState != null) {
+                _editTableETKey.currentState!.showOverlayOnTap();
+              }
+            }
+          },
+          key: _editTableETKey,
+          content: _editTableListView(false, context),
+          child: SizedBox(
+            width: widget.iconSize,
+            height: widget.iconSize,
+            child: Image.asset(
+              ImageConstant.kiEditTablePng,
+              color: widget.iconColor,
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _getEditTableItem(
+      EditTableEnum type, BuildContext context, bool isMobile) {
+    String value = ""
+
+    switch (type) {
+      case EditTableEnum.insertRowAbove:
+        value = "Insert Row Above";
+        imagePath = ImageConstant.kiInsertTablePng;
+        break;
+      case EditTableEnum.insertRowBelow:
+        value = "Insert Row Below";
+        imagePath = ImageConstant.kiInsertRowBelowPng;
+        break;
+      case EditTableEnum.deleteRow:
+        value = "Delete Row";
+        imagePath = ImageConstant.kiDeleteRowPng;
+        break;
+      case EditTableEnum.deleteColumn:
+        value = "Delete Column";
+        imagePath = ImageConstant.kiDeleteColumnPng;
+        break;
+      case EditTableEnum.insertColumnLeft:
+        value = "Insert Column Left";
+        imagePath = ImageConstant.kiInsertColumnLeftPng;
+        break;
+      case EditTableEnum.insertColumnRight:
+        value = "Insert Column Right";
+        imagePath = ImageConstant.kiInsertColumnRightPng;
+        break;
+      case EditTableEnum.deleteTable:
+        value = "Delete Table";
+        imagePath = ImageConstant.kiDeleteTablePng;
+        break;
+    }
+
+    return WebViewAware(
+      child: Card(
+        color: widget.dropDownColor,
+        child: InkWell(
+          child: Padding(
+            padding: const EdgeInsets.all(4.0),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.start,
+              mainAxisSize: MainAxisSize.min,
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                SizedBox(
+                    width: widget.iconSize,
+                    height: widget.iconSize,
+                    child: Image.asset(
+                      imagePath,
+                      color: widget.iconColor,
+                    )),
+                const SizedBox(
+                  width: 8,
+                ),
+                Expanded(
+                  child: Text(
+                    value,
+                    style: TextStyle(
+                        fontWeight: FontWeight.w500, color: widget.iconColor),
+                  ),
```

Comment:
**Suggestion:** Extract the TextStyle to a variable or constant to avoid duplication.
```suggestion
                  final TextStyle textStyle = TextStyle(fontWeight: FontWeight.w500, color: widget.iconColor);

                  Text(
                    value,
                    style: textStyle,
                  )
```